client/src/containers/explore/section/overview-section/index.tsx

```diff
@@ -7,7 +7,7 @@ import { isEmptyWidget } from "@/lib/utils";
 import TileMenu, { TileMenuItem } from "@/containers/explore/section/tile-menu";
 import NoData from "@/containers/no-data";
 import Widget from "@/containers/widget";
-import HorizontalBarChart from "@/containers/widget/horizontal-bar-chart";
+import HorizontalBarChart from "@/containers/widget/horizontal-bar-chart/explore";
```

Comment:
Why are we putting the chart in a `explore` folder? The chart should be totally agnostic of where it's being implemented.